tests/api/routes/gem2s.test.js

```diff
@@ -0,0 +1,179 @@
+const express = require('express');
+const request = require('supertest');
+const https = require('https');
+const _ = require('lodash');
+
+const logger = require('../../../src/utils/logging');
+const expressLoader = require('../../../src/loaders/express');
+const CacheSingleton = require('../../../src/cache');
+const gem2sResponse = require('../../../src/api/route-services/gem2s-response');
+const { createGem2SPipeline } = require('../../../src/api/general-services/pipeline-manage');
+
+jest.mock('sns-validator');
+jest.mock('aws-xray-sdk');
+jest.mock('../../../src/utils/authMiddlewares');
+jest.mock('../../../src/utils/logging');
+jest.mock('../../../src/cache');
+jest.mock('../../../src/api/route-services/gem2s-response');
+jest.mock('../../../src/api/general-services/pipeline-manage');
+jest.mock('../../../src/api/route-services/experiment');
+
+const basicMsg = JSON.stringify({
```

Comment:
is this `JSON.stringify` necessary? it seems like `basicMsg` is JSON.parsed later on when it's used (except for one test)

Reply:
No, you're right, I copied it from pipeline tests so I fixed it there too.